src/components/attendee/AttendeeLayout.module.scss

```diff
@@ -62,6 +63,11 @@ $-z-index--popover: layer.above($-z-index--ui);
 
 .LayerUiAbsolute {
   position: absolute;
+  top: 0px;
+
+  @media (min-width: breakpoint.$md) {
+    top: 12px;
```

Comment:
another hard-coded value in pixels

Reply:
[ce03ad0](https://github.com/sparkletown/sparkle/pull/2973/commits/ce03ad0c063b92163c5039c84f70f34635278d0b)